pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -111,122 +113,68 @@ private boolean isUnderDsmKey(String source) {
 
                 Participant participant = esDsm.getParticipant().orElse(new Participant());
 
+                List<ParticipantData> participantData = esDsm.getParticipantData();
+
+                ParticipantWrapperDto participantWrapperDto = new ParticipantWrapperDto();
+                participantWrapperDto.setEsData(elasticSearchParticipantDto);
+                participantWrapperDto.setParticipant(participant);
+                participantWrapperDto.setParticipantData(participantData);
+
+                List<MedicalRecord> medicalRecord = esDsm.getMedicalRecord();
+                participantWrapperDto.setMedicalRecords(medicalRecord);
+
                 esDsm.getOncHistory().ifPresent(oncHistory -> {
                     participant.setCreatedOncHistory(oncHistory.getCreated());
                     participant.setReviewedOncHistory(oncHistory.getReviewed());
                 });
 
-                List<MedicalRecord> medicalRecord = esDsm.getMedicalRecord();
-
                 List<OncHistoryDetail> oncHistoryDetails = esDsm.getOncHistoryDetail();
                 List<Tissue> tissues = esDsm.getTissue();
                 mapTissueToProperOncHistoryDetail(oncHistoryDetails, tissues);
-
-                List<KitRequestShipping> kitRequestShipping = esDsm.getKitRequestShipping();
-
-                List<String> proxyGuids = elasticSearchParticipantDto.getProxies();
-                String usersIndexES = ddpInstance.getUsersIndexES();
-                ElasticSearch participantsByIds = elasticSearchable.getParticipantsByIds(usersIndexES, proxyGuids);
-                List<ElasticSearchParticipantDto> proxies = participantsByIds.getEsParticipants();
-
-                List<ParticipantData> participantData = esDsm.getParticipantData();
-
-                ParticipantWrapperDto participantWrapperDto = new ParticipantWrapperDto();
-                participantWrapperDto.setEsData(elasticSearchParticipantDto);
-                participantWrapperDto.setParticipant(participant);
-                participantWrapperDto.setMedicalRecords(medicalRecord);
                 participantWrapperDto.setOncHistoryDetails(oncHistoryDetails);
-                participantWrapperDto.setKits(kitRequestShipping);
-                participantWrapperDto.setProxyData(proxies);
-                participantWrapperDto.setParticipantData(participantData);
+
                 participantWrapperDto.setAbstractionActivities(Collections.emptyList());
                 participantWrapperDto.setAbstractionSummary(Collections.emptyList());
 
-                result.add(participantWrapperDto);
+                List<KitRequestShipping> kitRequestShipping = esDsm.getKitRequestShipping();
+                participantWrapperDto.setKits(kitRequestShipping);
 
+                ESProfile esProfile = elasticSearchParticipantDto.getProfile().orElseThrow();
+                String ddpParticipantId = esProfile.getGuid();
+                if (proxiesByParticipantIds != null) {
```

Comment:
using the map of proxies here from the fetch… methods to not ask ES for user info again @gmakharat @Anzori-Ghurtchumelia